src/main/Config.h

```diff
@@ -420,6 +420,14 @@ class Config : public std::enable_shared_from_this<Config>
     // the entry cache
     size_t PREFETCH_BATCH_SIZE;
 
+    // If set to true, the application will halt when an internal error is
+    // encountered during applying a transaction. Otherwise, the
```

Comment:
we normally document external facing flags with the example config file

Reply:
Done. I still see the value of documenting this in code too, please let me know if you think this is unneeded.